sdk_lib/Dockerfile.sdk-import

```diff
@@ -38,6 +38,7 @@ RUN rm /home/sdk/.bashrc
 RUN cp /etc/skel/.bashrc /home/sdk
 RUN echo "cd /home/sdk/trunk/src/scripts" >> /home/sdk/.bashrc
 RUN echo 'export PATH="$PATH:/usr/local/bin:/usr/local/sbin"' >> /home/sdk/.bashrc
+RUN echo "if [-f /etc/profile.d/qemu-aarch64.sh ]; then source /etc/profile.d/qemu-aarch64.sh; fi" >> /home/sdk/.bashrc
```

Comment:
Is this the only file we anticipate being there? If not then a for loop over the files in `/etc/profile.d` makes sense.

Reply:
We could source `/etc/profile` which already loops over the files in `/etc/profile.d` but maybe we get a circle…? No idea why `/etc/profile` is skipped now.

Reply:
As found out it, we need `-l` for bash and su